modules/plugin/geopkg/src/test/java/org/geotools/geopkg/GeoPkgDataStoreFactoryTest.java

```diff
@@ -54,7 +137,6 @@ public void testBaseDirectory() throws IOException {
         DataStore data = factory.createDataStore(map);
         data.createSchema(b.buildFeatureType());
         data.dispose();
-
-        assertTrue(new File(tmp.getRoot(), "foo.gpkg").exists());
+        return data;
```

Comment:
"data" here is dead, it has been disposed of. It's also not used anywhere I can see.
So I'd switch the method to void.

Reply:
fixed